big_tests/tests/instrument_cets_SUITE.erl

```diff
@@ -0,0 +1,46 @@
+-module(instrument_cets_SUITE).
+-include_lib("eunit/include/eunit.hrl").
+
+-compile([export_all, nowarn_export_all]).
+
+-import(distributed_helper, [mim/0, mim2/0, rpc/4]).
+-import(domain_helper, [host_type/1]).
+-import(mongooseimctl_helper, [rpc_call/3]).
+
+all() ->
+    [{group, instrument}].
+
+groups() ->
+    [{instrument, [], cases()}].
+
+cases() ->
+    [check_instrumentation].
+
+init_per_suite(Config) ->
+    case rpc_call(mongoose_config, lookup_opt, [[internal_databases, cets]]) of
+        {error, not_found} ->
+            {skip, "CETS is not configured"};
+        {ok, _} ->
+            instrument_helper:start(instrument_helper:declared_events(mongoose_metrics_probe_cets)),
+            mongoose_helper:backup_and_set_config_option(Config, [instrumentation, probe_interval], 1)
```

Comment:
This doesn't work, because the probe timer is already running. You would need to restart the probe for this to take effect. Alternatively, we could set `instrumentation.probe_interval` to 1 in the TOML vars for `mim1`. In your case the interval is the default 15 seconds.